src/vaishnaviVJun20_arrayExamples/FirstDifferIndex.java

```diff
@@ -0,0 +1,19 @@
+package vaishnaviVJun20_arrayExamples;
+
+public class FirstDifferIndex {
+	
+	void findFirstDifferIndex(int[] array1, int[] array2) {
+	   for(int index=0;index<array1.length;index++) {
+		   if(array1[index]!= array2[index])
+			   System.out.println("Values are not matching at index->" +index);
```

Comment:
Where is the return statement ?

Reply:
I have added the return statement. Got the logic. Thanks for the suggestion.